managed-services/transactional-email/how-to/add-dmarc-record.mdx

```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: How to configure a DMARC record
+  description: This page explains how to configure a DMARC record
+content:
+  h1: How to configure a DMARC record
+  paragraph: This page explains how to configure a DMARC record
+tags: spf spf-record dkim dkim-records transactional
+dates:
+  validation: 2024-02-21
+  posted: 2024-02-21
+categories:
+  - managed-services
+---
+
+This page shows you how to configure a [DMARC record](/managed-services/transactional-email/concepts/#dmarc-record) on your domain from the **Scaleway console**.
+If you have registered a domain at another provider, you can still use this documentation, as the process is similar.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/).
+  - You have a domain name.
+  - You have [configured your domain with Transactional Email](/managed-services/transactional-email/how-to/configure-domain-with-transactional-email).
+  - You have configured SPF and DKIM records on your domain.
```

Comment:
The DMARC record will be added as long as SPF and DKIM records. I think we can remove this line.